app/src/main/res/layout-land/profile_edit_fragment.xml

```diff
@@ -54,6 +54,7 @@
         android:layout_marginStart="24dp"
         android:importantForAccessibility="no"
         android:text="@{viewModel.profile.name}"
+        android:textColor="@color/component_color_profile_edit_activity_profile_name_text_viewer_color"
```

Comment:
Rename to `component_color_profile_edit_activity_profile_edit_name_text_view_color `

Reply:
done